README.old.md

```diff
@@ -0,0 +1 @@
+# react-redux-capstone
```

Comment:
- [ ] It would be better to add the bar that the previous reviewer asked for in-home and details page  in order to match the design 
- Yours 
<img width="285" alt="Screenshot 2023-04-28 at 14 59 32" src="https://user-images.githubusercontent.com/61790587/235168530-a7dbcba0-0e3a-4280-8ff4-08371726922e.png">
<img width="352" alt="Screenshot 2023-04-28 at 14 59 25" src="https://user-images.githubusercontent.com/61790587/235168534-c51936f0-ea7b-4579-8ed5-a05329057593.png">
- Design 
<img width="384" alt="Screenshot 2023-04-28 at 14 58 57" src="https://user-images.githubusercontent.com/61790587/235168561-207ae7cf-76b3-4ae5-9812-c5c77f0f248a.png">
<img width="385" alt="Screenshot 2023-04-28 at 14 59 07" src="https://user-images.githubusercontent.com/61790587/235168567-4842d16f-3da7-4d49-8712-855137cba94b.png">

Reply:
This change has been done well